src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -267,6 +269,7 @@ private static void AddExceptionTags(Exception exception, Activity activity, out
 
             if (tracingEmitNewAttributes)
             {
+                activity.SetTag(SemanticConventions.AttributeNetworkProtocolVersion, HttpTagHelper.GetProtocolVersionString(protocolVersion));
```

Comment:
Is this meant to be set twice?
It's set here (line 272) in the `AddExceptionTags` and also the out parameter is used to set it again at Line 543.

It's also odd to me that it's being set in the AddExceptionTags helper method. This may be a candidate for refactor in the future.

Reply:
line 543 sets it on metric.